src/lib/error-custom.ts

```diff
@@ -125,7 +125,7 @@ class ErrorCustom extends Error {
 
     // Index Name
     const serviceName = process.env.ELASTIC_LOGGING_SERVICE_NAME || 'error-custom';
-    const indexName = process.env.ELASTIC_LOGGING_INDEX || `logs-${serviceName}-${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
+    const indexName = process.env.ELASTIC_LOGGING_INDEX || `logs-${serviceName}-${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`;
```

Comment:
This doesn't fit the required schema.
The required schema is YYYY-MM-dd; this doesn't pad the dates.

Replace with `const indexName = process.env.ELASTIC_LOGGING_INDEX || `logs-${serviceName}-${date.toISOString().split('T').shift()}`

Reply:
If we used the ISO format it'd also include the time code, which could mean an index per request